### ISSUE TITLE
Publisher Deal Types need full coverage in the use case set 

### ISSUE THREAD

**Issue by AramZS** [w3c/web-advertising#111]
Right now we really only discuss (and only vaguely) direct sold vs programmatic among our advertising use cases. We really need to document a few additional things:

- The nature and flow of direct sold deals and the different way they are implemented by publishers 
- The other types of deals between programmatic and direct; including Private Marketplaces, Programmatic Direct and other types of sales that fall between the two poles of 100% hands on and 0% hands on by publishers. 
- The reasoning why these different models exist and how, why or if they should be preserved in a privacy-focused future. 
- How bidding systems work to compare multiple bids from multiple providers, why they do so, and to what scale and end purpose and the impact of that for both buy and sell side. 
- We've previously discussed bid shading and auction models, having more detail in our use case document about bid models and auction models would be very useful.

This is a lot about publisher models so their participation is important here, and this also comprises multiple things that likely will have to be spun into multiple issues. Consider this a sort of omnibus issue to talk about this.

**Comment by rblanck** [comment]
We support this request. The impact into Publisher business modells and their actual proccess inside the advertising infrastructure, especially in countries which have a strong direct business, are not fully discussed, although many of these proposals are intervening the current (direct) business flows.

**Comment by peligio** [comment]
We are happy to support this and contribute more detailed work around it to help surface publisher needs with more clarity.

There are various strands of monetisation within these advertising sale types which may be impacted in varying ways by emergent proposals. Furthermore, this work is important to illustrate why these use cases are important to continue to support.

**Comment by darobin** [comment]
Agreed, we can't solve advertising on the Web without addressing this.

**Comment by dmarti** [comment]
At a previous meeting we went over some requirements for lower-level functionality that helps enable various publisher ad models.

 * [Placement](https://github.com/dmarti/in-browser-auction-publisher-issues/blob/main/placement.md)

 * [Reporting](https://github.com/dmarti/in-browser-auction-publisher-issues/blob/main/reporting.md)
 
Those are focused on specific features, such as "log and report that a problem ad did not appear on a site after a publisher blocked it" and "do not show ads for direct competitors on the same page.

I am looking forward to discussing these issues from the type of deal point of view (top down in addition to bottom up) and will update as needed.

**Comment by TheMaskMaker** [comment]
@AramZS Amazing issue. 100% agree these things need more attention. This is I believe (someone tell me if I'm wrong or there is another list) the list referenced in the W3 meeting  https://github.com/w3c/web-advertising/blob/main/support_for_advertising_use_cases.md  to add publisher use cases. I've also been compiling my own publisher concern list and hope to make PRs soon.

**Comment by peligio** [comment]
I was looking to break down the needs for this 'backgrounder' document mentioned in the first post by @AramZS and the below outline is what the ask looks like so far. A couple of questions for other contributors:

1. This looks a lot like something that's been written many times before elsewhere (and internally at publishers), so is this something that can be sourced from an existing place or linked to?
2. Is there any more detail/depth that would be interesting to add?

From this outline, then there would be more specific guidance on use cases, or further emphasis on existing ones.

===

## Publisher monetisation - Ad Sales ##

**Advertising Sales Types**

- Direct
  - Why does it exist?
  - Importance moving forward

- Programmatic
  - Why does it exist?
  - Importance moving forward
  - Deal Types
    - Open Marketplace (OMP)
    - Private Marketplaces (PMP)
    - Programmatic Direct/Guaranteed (PD/PG)
  - Auction bidding types
     - First price
        - Include note on ‘bid shading’
     - Second price

**How bidding systems work in a multiple bidder & multiple provider system**

- Why the system works like this? (Including simple diagram)
- What is the impact on both buy- and sell-sides?
